lib/rls/api.rb

```diff
@@ -122,6 +129,80 @@ def seasons(renew = false)
       end
     end
 
+    # Retrieve the current season's tiers.
+    # [API docs for this method](http://documentation.rocketleaguestats.com/#tiers)
+    # @param renew [true, false] Ignore the cache and make a new request
+    # @return [Hash<Integer => Tier] The tiers by ID
+    def tiers(renew = false)
+      if renew || !@tiers
+        response =
+          request(
+            :get,
+            'data/tiers'
+          )
+        @tiers =
+          response.map { |data| [data['tierId'], Tier.new(data)] }.to_h
+      else
+        @tiers
+      end
+    end
+
+    # Retrieve the current playlists.
+    # [API docs for this method](http://documentation.rocketleaguestats.com/#playlists)
+    # @param renew [true, false] Ignore the cache and make a new request
+    # @return [Hash<Integer => Playlist] The playlist by ID
+    def playlists(renew = false)
+      if renew || !@playlists
+        response =
+          request(
+            :get,
+            'data/playlists'
+          )
+        @playlists =
+          response.map { |data| [data['id'], Playlist.new(data)] }.to_h
+      else
+        @playlists
+      end
+    end
+
+    # Returns the top 100 players for the current season on a particular playlist.
+    # [API docs for this method](http://documentation.rocketleaguestats.com/#ranked-leaderboard)
+    # @param playlist_id [Integer]
+    # @return [Array<Player>]
+    def ranked_leaderboard(playlist_id)
+      raise ArgumentError, 'Invalid playlist_id' unless playlists.keys.include?(playlist_id)
```

Comment:
FWIW: This does indeed make this the first endpoint that executes another API request (potentially) but I'd rather do this than send a bad request. Let me know if you think otherwise.